stripe2qbo/api/routers/transaction_router.py

```diff
@@ -0,0 +1,43 @@
+from typing import Annotated
+
+from sqlalchemy.orm import Session
+from fastapi import APIRouter, HTTPException, Depends
+
+from stripe2qbo.api.dependencies import get_current_user, get_db
+from stripe2qbo.db.models import TransactionSync, User
+
+router = APIRouter(
+    prefix="/transaction",
+    tags=["transaction"],
+)
+
+
+@router.get("/")
+async def get_all_transactions(
+    db: Annotated[Session, Depends(get_db)],
+    user: Annotated[User, Depends(get_current_user)],
+) -> TransactionSync:
```

Comment:
not sure you can use SQLAlchemyModels as FastAPI response models. You can check by running the server and see if the docs compile

This is why there are Pydantic models defined in `db/schemas`

I'd suggest moving the TransactionSycn model from sync.py there and using it as a response model here

Reply:
This still needs addressing